README.md

```diff
@@ -260,6 +260,7 @@ Refer  below for next steps when any indicator reports a `false`.
 | S0017        | Determines if there are any `OutOfMemory` errors in the `puppetdb` log. | [Increase the Java heap size for that service.](https://support.puppet.com/hc/en-us/articles/360015511413) | Open a Support ticket referencing S0017 and provide [puppet metrics](https://support.puppet.com/hc/en-us/articles/231751308), `/var/log/puppetlabs/puppetdb/puppetdb.log`, and the output of `puppet infra tune`.|
 | S0018        | Determines if there are any `OutOfMemory` errors in the `orchestrator` log. | [Increase the Java heap size for that service.](https://support.puppet.com/hc/en-us/articles/360015511413) | Open a Support ticket referencing S0018 and provide [puppet metrics](https://support.puppet.com/hc/en-us/articles/231751308), `/var/log/puppetlabs/orchestration-services/orchestration-services.log`, and output of `puppet infra tune`.|
 |S0019|Determines if there are sufficent jRubies available to serve agents.| Insufficient jRuby availability results in queued puppet agents and overall poor system performance. There can be many causes: [Insufficient server tuning for load](https://support.puppet.com/hc/en-us/articles/360013148854), [a thundering herd](https://support.puppet.com/hc/en-us/articles/215729277), and [insufficient system resources for scale.](https://puppet.com/docs/pe/latest/hardware_requirements.html#hardware_requirements) | If self-sevice fails to resolve the issue, open a ticket referencing S0019 and provide a description of actions so far and the output of the [support script.](https://puppet.com/docs/pe/latest/getting_support_for_pe.html#pe_support_script)
+| S0020        | Determines if the Console status api reports all services as running | Checks Infrastracture services API for classifier information - Distinct to S0004. Which ever service returns in a state that is not running, examine the logging for that service to indicate the fault. |
```

Comment:
This table has 4 columns, but you only have 3 here?

Indicator ID | Description | Self-service steps | What to include in a Support ticket

You have the id, and the description, the self-service steps has a second description which conflicts with the first "Checks Infrastracture services API for classifier information - Distinct to S0004." and you dont have a what to include in the support section, i would do a markdownlink to the api you are checking, and a link to the docs for where the logs are when you mention them, and perhaps tell the customer to send a description of the service thats failing as well.

The aim here is to give the customer the tools to react to the test failure, and where they fail to interpret the data, send it to us with a description of the fault